internal/sdkv2provider/resource_cloudflare_teams_accounts_test.go

```diff
@@ -109,8 +111,12 @@ resource "cloudflare_zero_trust_gateway_settings" "%[1]s" {
   proxy {
     tcp = true
     udp = false
-	root_ca = true
-	virtual_ip = true
+    root_ca = true
+    virtual_ip = true
+  }
+  connectivity {
```

Comment:
this still has the same test failure from the previous PR that hasn't been addressed by the service team yet.

```
TF_ACC=1 go test ./internal/sdkv2provider -v -run "^TestAccCloudflareTeamsAccounts_" -count 1 -timeout 120m -parallel 1
=== RUN   TestAccCloudflareTeamsAccounts_ConfigurationBasic
    resource_cloudflare_teams_accounts_test.go:23: Step 1/1 error: Error running apply: exit status 1
        
        Error: error updating Teams Account connectivity settings for account "f037e56e89293a057740de681ac9abbe": error unmarshalling the JSON response error body: invalid character '' looking for beginning of value
        
          with cloudflare_zero_trust_gateway_settings.opwxnpfurs,
          on terraform_plugin_test.tf line 12, in resource "cloudflare_zero_trust_gateway_settings" "opwxnpfurs":
          12: resource "cloudflare_zero_trust_gateway_settings" "opwxnpfurs" {
        
--- FAIL: TestAccCloudflareTeamsAccounts_ConfigurationBasic (2.39s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	3.476s
FAIL
make: *** [testacc] Error 1
```